example/listenvironments/main.go

```diff
@@ -0,0 +1,49 @@
+package main
```

Comment:
Please add the standard header used in this repo, with a blank line, followed by a godoc comment like this:

```go
// listenvironments is an example of...
```

Reply:
Done ✅